<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Concept</class>
 <widget class="QMainWindow" name="Concept">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1425</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>288</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="btnOpenFile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="leftDisplay" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4"/>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_7" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="8" column="0">
              <widget class="QLabel" name="label_28">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Lobo Inferior</string>
               </property>
              </widget>
             </item>
             <item row="8" column="2">
              <widget class="QLabel" name="label_29">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>(89.256cm³)</string>
               </property>
              </widget>
             </item>
             <item row="5" column="5">
              <widget class="QLabel" name="label_37">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>(140.356cm³)</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3" colspan="2">
              <widget class="QCheckBox" name="checkBox_4">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>Pavimentação em mosaico</string>
               </property>
              </widget>
             </item>
             <item row="7" column="5">
              <widget class="QLabel" name="label_39">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>(140.356cm³)</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3" colspan="2">
              <widget class="QCheckBox" name="checkBox_5">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>Bilateralidade</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QLabel" name="label_23">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>(0.000cm³)</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0" colspan="2">
              <widget class="QCheckBox" name="checkBox_3">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>Listras Fibrosas</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color:  rgb(204, 0, 0);</string>
               </property>
               <property name="text">
                <string>94.62%</string>
               </property>
              </widget>
             </item>
             <item row="7" column="2">
              <widget class="QLabel" name="label_26">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>(0.000cm³)</string>
               </property>
              </widget>
             </item>
             <item row="5" column="4">
              <widget class="QLabel" name="label_8">
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>7.26%</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QCheckBox" name="checkBox">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>Opacidades de Vidro-Fosco</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QLabel" name="label_27">
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>6.52%</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Possibilidade de NCP:</string>
               </property>
              </widget>
             </item>
             <item row="8" column="5">
              <widget class="QLabel" name="label_34">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>(0.000cm³)</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_25">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Lobo Central</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Pulmão Direito:</string>
               </property>
              </widget>
             </item>
             <item row="7" column="3">
              <widget class="QLabel" name="label_36">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Lobo Central</string>
               </property>
              </widget>
             </item>
             <item row="8" column="4">
              <widget class="QLabel" name="label_32">
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>0.00%</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_21">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Lobo Superior</string>
               </property>
              </widget>
             </item>
             <item row="6" column="3">
              <widget class="QLabel" name="label_35">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Lobo Superior</string>
               </property>
              </widget>
             </item>
             <item row="8" column="3">
              <widget class="QLabel" name="label_33">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Lobo Inferior</string>
               </property>
              </widget>
             </item>
             <item row="6" column="4">
              <widget class="QLabel" name="label_31">
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>0.00%</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Pulmão:</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLabel" name="label_22">
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>0.00%</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLabel" name="label_10">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>(89.256cm³)</string>
               </property>
              </widget>
             </item>
             <item row="4" column="3" colspan="2">
              <widget class="QCheckBox" name="checkBox_6">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>Outros</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_41">
               <property name="text">
                <string>(229.612cm³)</string>
               </property>
              </widget>
             </item>
             <item row="5" column="3">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Pulmão Esquerdo:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_40">
               <property name="font">
                <font>
                 <pointsize>18</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>13.81%</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QCheckBox" name="checkBox_2">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>Consolidações com OVF</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="label_6">
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>6.52%</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLabel" name="label_24">
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>0.00%</string>
               </property>
              </widget>
             </item>
             <item row="6" column="5">
              <widget class="QLabel" name="label_38">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="text">
                <string>(0.000cm³)</string>
               </property>
              </widget>
             </item>
             <item row="7" column="4">
              <widget class="QLabel" name="label_30">
               <property name="styleSheet">
                <string notr="true">color: rgb(52, 101, 164);</string>
               </property>
               <property name="text">
                <string>7.26%</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="MainDisplay" name="centerDisplay" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="chartTab">
             <attribute name="title">
              <string>Gráficos</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_3"/>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainDisplay</class>
   <extends>QWidget</extends>
   <header>MainDisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnOpenFile</sender>
   <signal>clicked()</signal>
   <receiver>Concept</receiver>
   <slot>openFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFileDialog()</slot>
 </slots>
</ui>
